checkov/common/bridgecrew/platform_integration.py

```diff
@@ -249,6 +249,30 @@ def is_bc_token(token: str | None) -> TypeGuard[str]:
 
         return re.match(UUID_V4_PATTERN, token) is not None
 
+    @staticmethod
+    def raise_bridgecrew_auth_error(request_status: int, request_data: bytes) -> NoReturn:
+        logging.error(f'Received {request_status} response from Prisma /login endpoint: {request_data.decode("utf8")}')  # danger:ignore
+        raise BridgecrewAuthError()
+
+    def fetch_auth_token(self, username: str, password: str) -> str:
+        retries = int(os.getenv('REQUEST_MAX_TRIES', 3))
+        request: Any = None
+        for i in range(retries):
+            request = self.http.request("POST", f"{self.prisma_api_url}/login",  # type:ignore[union-attr]
+                                        body=json.dumps({"username": username, "password": password}),
+                                        headers=merge_dicts({"Content-Type": "application/json"},
+                                                            get_user_agent_header()))
+
+            if request.status == 200:
```

Comment:
we can use `request.ok` instead so we are not tied directly to 200, if there are any future changes

Reply:
Are you sure that this attribute exists?